pkg/flow/ent/schema/instance.go

```diff
@@ -36,10 +37,12 @@ func (Instance) Edges() []ent.Edge {
 		edge.From("namespace", Namespace.Type).Ref("instances").Required().Unique().Annotations(entsql.Annotation{OnDelete: entsql.Cascade}),
 		edge.From("workflow", Workflow.Type).Ref("instances").Unique().Annotations(entsql.Annotation{OnDelete: entsql.SetNull}),
 		edge.From("revision", Revision.Type).Ref("instances").Unique().Annotations(entsql.Annotation{OnDelete: entsql.SetNull}),
+		edge.To("ins", Instance.Type).From("orginator").Required().Unique(),
```

Comment:
I think this information is already available to us without requiring us to modify the database.